scripts/vsc-venv-wrapper/README.md

```diff
@@ -0,0 +1,82 @@
+# Python Virtual Environment Wrapper Script
+
+`vsc-venv` is a script that encapsulates the creation and management of Python virtual environments. 
+This avoids multiple issues with the default `venv` command on the HPC. 
+
+
+One issue is that a virtual environment created for one cluster might not work on another. 
+Additionally, when activating a virtual environment, 
+the same centrally installed modules used during its creation must also be loaded.
+the `vsc-venv` command manages multiple virtual environments for different clusters in a transparent way, 
+while guaranteeing the same module environment.
+
+## Usage
+
+A virtual environment can be activated by running the following command:
+
+```bash
+source ./vsc-venv.sh activate <requirements> [module_script]
+```
+
+Here, `requirements` is the path to a file containing the python dependencies to install in the virtual environment.
+An optional module script can be provided to load the required modules before creating the virtual environment.
```

Comment:
i don't get the optional part. if it is missing, the system python is being used, no?

Reply:
Yes, and perhaps that should come with a warning.

There are use cases where using system Python is OK. EasyBuild for example, or ReFrame.

Reply:
if the venv does not exist, it should be mandatory and stored after venv creation

if the venv exists, it should be optional. if is not provided, reuse the one that was used during creation. if it is provided and different from original one, hmmm. a bit tricky; but you could overwrite the original one?

so for system python you can use an empty file or eg pass the 'SYSTEM' magic constant (but i would just do empty file)

Reply:
Fixed in [warning when system python is used + logging of python version](https://github.com/hpcugent/vsc_user_docs/pull/670/commits/3b924f433e971bde5ca1ebf458e55b9ecb08ccc2)